knowledge_gpt/main.py

```diff
@@ -27,12 +29,18 @@
 st.set_page_config(page_title="KnowledgeGPT", page_icon="📖", layout="wide")
 st.header("📖KnowledgeGPT")
 
+# Initialising submit as False
```

Comment:
Lets remove all code comments that just describe what you are doing. In production code, we don't really use comments. Python is a nice readable language, so clean code is understandable in most cases by other devs. Comments in code are reserved for when you really want to explicitly tell other devs why you have coded things in a certain way.